sql/sql_priv.h

```diff
@@ -354,6 +354,7 @@
   data dictionary without changing table rows
 */
 #define IS_EQUAL_PACK_LENGTH 2
+#define IS_EQUAL_BUT_CHARSET 3
```

Comment:
There is some hint that these could be treated as bitmasks. Should the value be 4 instead of 3? Can you add a `/** comment */` before it?